templates/zerver/create_demo_realm.html

```diff
@@ -0,0 +1,50 @@
+{% extends "zerver/portico_signup.html" %}
+
+{% block title %}
+<title>{{ _("Try Zulip in a demo organization") }} | Zulip</title>
+{% endblock %}
+
+{% block portico_content %}
+<div class="register-account flex full-page">
+    <div class="center-block new-style" id="demo-realm-creation">
+        <div class="pitch">
+            <h1>{{ _("Try Zulip in a demo organization") }}</h1>
+            <p>{{ _("No email required.") }}</p>
+        </div>
+        <form class="white-box" id="create_demo_realm" action="{{ current_url() }}" method="post">
+            {{ csrf_input }}
+            <input type='hidden' name='timezone' id='demo-creator-timezone'/>
+            {% include 'zerver/realm_creation_base_form_fields.html' %}
+            {% include 'zerver/found_zulip_form_field.html' %}
+            <div class="terms-of-service">
+                {% if terms_of_service %}
+                <div class="input-group">
+                    <label for="id_terms" class="inline-block checkbox">
+                        <input id="id_terms" class="required" type="checkbox" name="terms"
+                          {% if form.terms.value() %}checked="checked"{% endif %} />
+                        <span class="rendered-checkbox"></span>
+                        {% trans %}I agree to the <a href="{{ root_domain_url }}/policies/terms" target="_blank" rel="noopener noreferrer">Terms of Service</a>.{% endtrans %}
+                    </label>
+                    {% if form.terms.errors %}
+                        {% for error in form.terms.errors %}
+                        <p class="error help-inline alert alert-error">{{ error }}</p>
+                        {% endfor %}
+                    {% endif %}
+                </div>
+                {% endif %}
+            </div>
```

Comment:
Could this ToS input-group also be extracted as a template and shared?

Reply:
Done.